### PR TITLE
Add build and push image CI

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

Enhance https://github.com/fluent/fluentbit-operator/pull/210

Push images to the org like kubesphere after adding accounts with permissions. Suppose the tag is latest.

- kubepshere/fluentbit-operator:latest with buildx(and64/arm64)
- kubesphere/fluentbit:latest with amd64
- kubesphere/fluentd:latest with amd64

Each pr after merged will push a new image for fluentbit-operator:latest.
If the pointed files changed, the workflow of fluentbit/fluentd image will be triggered to build. 

/cc @patrick-stephens @benjaminhuo @wanjunlei @wenchajun Pls move here.

### CODE REVIEW

**Comment by zhu733756** [comment]
See https://github.com/zhu733756/fluentbit-operator/actions/runs/1880252411 and https://github.com/zhu733756/fluentbit-operator/actions/runs/1880252403

File: .github/workflows/build-fb-image.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        with:
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Setting Environment Variables
```
Comment (wanjunlei) [inline .github/workflows/build-fb-image.yaml:44]: This should be deleted.

File: .github/workflows/build-fd-image.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Setting Environment Variables
+        run: |
```
Comment (wanjunlei) [inline .github/workflows/build-fd-image.yaml:45]: This should be deleted.

File: .github/workflows/build-op-image.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+name: Building Fluent Operator image
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+    paths:
```
Comment (wanjunlei) [inline .github/workflows/build-op-image.yaml:9]: Need to exclude the directory `pkg/fsnotify`

**Review by wanjunlei — CHANGES_REQUESTED** [review]

**Review by wanjunlei — APPROVED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**